src/script/auth/Environment.ts

```diff
@@ -27,9 +27,11 @@ export enum ENVIRONMENT {
   PRODUCTION = 'PRODUCTION',
 }
 
-export const APP_ENVIRONMENT = getEnvironmentFromQuery();
-checkEnvironment();
-export function getEnvironmentFromQuery() {
+export function isLocalhost() {
+  return window.location.hostname.includes('localhost') || window.location.hostname.startsWith('192.168.');
```

Comment:
localhost could also be `127.0.0.1` or another IP..

Reply:
I don't think we need to change this behavior right now. This will be completely removed in the near future